scripts/lbnl_time.nhc

```diff
@@ -0,0 +1,31 @@
+# LBNL NHC - Time checks
+#
+# Andy Georges <andy.georges@ugent.be>
+# 15 December 2010
+#
+
+
+# Checks the drift from the ptpd status file and compares it to the given threshold
+function check_ptpd_drift() {
+    local DRIFT=0
+
+    if [ ! -f "$NHC_PTPD_STATUS" ]; then
+        die 1 "$FUNCNAME: No $NHC_PTPD_STATUS file found"
+        return 1
+    fi
+    DRIFT=$(grep "Offset from Master" < "$NHC_PTPD_STATUS" | tr -s " " | cut -d" " -f5)
+    dbg "$FUNCNAME: drift is $DRIFT"
+
+    if [ -z "${DRIFT+x}" ]; then
```

Comment:
more unknown bash. why not `-z "$DRIFT"`?

Reply:
No, this is the right way. Otherwise, you cannot distinguished between unset and ""-valued variables. See also https://stackoverflow.com/a/13864829/349809 :)